br/pkg/lightning/backend/tidb/tidb.go

```diff
@@ -141,10 +141,11 @@ func NewTargetInfoGetter(db *sql.DB) backend.TargetInfoGetter {
 // TODO: refactor
 func (b *targetInfoGetter) FetchRemoteTableModels(ctx context.Context, schemaName string) ([]*model.TableInfo, error) {
```

Comment:
In the lightning logic, is it easy to pass the table-level allow list to this function, so we don't need to care about unrelated tables?

No need to change in this PR

Reply:
Can you check if this idea will work? But we don't need to change this PR